src/ShapeCrawler/ShapeCollection/ShapeGeometry.cs

```diff
@@ -105,33 +131,53 @@ public decimal CornerSize
     {
         get
         {
-            var shapeType = this.APresetGeometry?.Preset?.Value;
-
-            if (shapeType == A.ShapeTypeValues.RoundRectangle)
+            var adjustments = this.Adjustments;
+            return (this.GeometryType, adjustments.Length) switch
             {
-                return this.ExtractCornerSizeFromRoundRectangle();
-            }
-
-            if (shapeType == A.ShapeTypeValues.Round2SameRectangle)
-            {
-                return this.ExtractCornerSizeFromRound2SameRectangle();
-            }
-
-            return 0;
+                (Geometry.RoundedRectangle, 0) or 
+                (Geometry.TopCornersRoundedRectangle, 0) => 35,
```

Comment:
If `35` is the defaultze, then please add comment like `// default corner size predefined in PowerPoint` to avoid being it a magic number.

Reply:
0dbde615f47d0052cd67d33db03534162af53760